### ISSUE TITLE
⌨ Notification delete button confusing on VoiceOver

### ISSUE THREAD

**Issue by yonikid15** [GCTC-NTGC/gc-digital-talent#16012]
### ⌨ Accessibility and Expected Behaviour

The delete notification button seems to announce that it's a collapsible/expandable item. It should be only be announcing that its a modal.

### 📋 Steps to Reproduce

1. Open notification menu
2. Navigate to a notification
3. Open drop down and move to delete button item

### What operating systems are you encountering the problem on?

iOS

### What browsers are you encountering the problem on?

Safari

### What assistive technologies are you encountering the problem on?

VoiceOver

### ✅ Acceptance Criteria

- [ ] Notification delete button only announces that it's modal
